packages/volto/src/helpers/User/User.js

```diff
@@ -12,3 +12,32 @@ import { intersection } from 'lodash';
 export function userHasRoles(user, roles = []) {
   return intersection(user?.roles ?? [], roles).length > 0;
 }
+
+/**
+ * Checks if the user is Manager.
+ * @method isManager
+ * @returns {boolean}
+ */
+export function isManager(user) {
+  return userHasRoles(user, ['Manager']);
```

Comment:
I forget, do the roles in the user serialization include roles that the user has because of membership in a group?

Reply:
I confirmed it includes roles that come from group memberships.